content/teaching-curriculum/chapter1-high-standards/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+content_type: project
+tags:
+- teaching
+prerequisites:
+  hard:
+  - teaching_curriculum/project_quick_familiarity_1
+ready: true
+submission_type: link
+title: Chapter 1 - High Standards
+---
+
+Here is the [12 page book summary](https://www.boyd.k12.ky.us/userfiles/496/Classes/27400/Teach%20Like%20A%20Champion%20-%20The%20Main%20Idea.pdf) which you'd have seen in the quick familiarity project(s). It might be useful if you need a very brief refresher on the various techniques. Note that the scope of focus for this whole course encompasses everything that we do as teachers at Umuzi, including Code Reviews, Technical Sessions, learner support and any other interactions we might have with learners. 
+
+In this project, we're going to hone in on the first chapter, which is focused on techniques that set and uphold high academic standards. 
+
+These are the 5 techniques covered in the chapter: 
+1. No Opt Out
+2. Right is right
+3. Stretch it
+4. Format matters
+5. Without apology
+
+## Your Mission Objectives
+1. For each of these 5 techniques, reflect on your own experiences as a learner (before Umuzi), and describe a teacher you can remember who applied the technique really well, or really poorly in your classroom. 
+2. Now reflect on your time at Umuzi, both as a learner and a teacher. For each of these 5 techniques, describe how/where you've seen it applied really well. In a separate paragraph, describe how/where you've seen it done really poorly. 
+3. What 1 system/change do you think Umuzi should put in place to improve how we uphold high academic standards? Describe your suggested system/change in at least 3, but no more than 6 short paragraphs. 
+
+Answer each of the above objectives in no more than 300 words. 
+Do this on a Google doc and submit a link to the doc. MAKE SURE THAT THE SHARING PERMISSIONS ALLOW ANYONE ON THE INTERNET TO VIEW AND COMMENT ON THE FILE! 
```

Comment:
`AND COMMENT ON THE FILE! ` - i do not believe this is possible, as far as i can see you can allow anyone to view but not comment on a doc

Reply:
I edited this on each file